ui/src/importer/importerTaskDetails/ImportedSearch.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Search, Button, Icon } from 'semantic-ui-react';
+import debounce from 'lodash/debounce';
+
+export class ImportedSearch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.searchBar = React.createRef();
+  }
+
+  componentDidUpdate(prevProps) {
+    const { records } = this.props;
+    if (prevProps.records !== records) {
```

Comment:
As already chat, please create a `prop` `onSearchChange: func` and let the caller use this function when needed. Do not compare props values prev/current if not really needed, it can lead to nasty bugs.